cmd/promtool/unittest_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Prometheus Authors
```

Comment:
There is already a PR for unit tests for unit test https://github.com/prometheus/prometheus/pull/6062, I would like to avoid duplication of effort. Can you add these test cases to that PR instead?

Reply:
Or more like it would be better if we merge that PR first and merge this with the additional test case.

Reply:
I'm fine with this PR going after #6062 is merged. I had created the one basic test to prove that deserialization of durations such as `1d` was now working

Reply:
The other PR seems to have stalled, so I'll just go ahead and merge this.